numpydoc/docscrape_sphinx.py

```diff
@@ -143,16 +194,12 @@ def _str_member_list(self, name):
                 out += [''] + autosum
 
             if others:
-                maxlen_0 = max(3, max([len(x[0]) + 4 for x in others]))
-                hdr = sixu("=") * maxlen_0 + sixu("  ") + sixu("=") * 10
-                fmt = sixu('%%%ds  %%s  ') % (maxlen_0,)
-                out += ['', '', hdr]
+                table = []
                 for param, param_type, desc in others:
-                    desc = sixu(" ").join(x.strip() for x in desc).strip()
                     if param_type:
-                        desc = "(%s) %s" % (param_type, desc)
-                    out += [fmt % ("**" + param.strip() + "**", desc)]
-                out += [hdr]
+                        desc = [param_type, ''] + desc
+                    table.append(["**" + param.strip() + "**", desc])
+                out += ['', ''] + _grid_table_rst(table)
```

Comment:
What's the reason to use grid tables? simple tables are supposed to support multi-line rows:
http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#tables

(unless that might be too fragile?)